piston/utils/helpers.py

```diff
@@ -87,3 +89,22 @@ def set_style(theme: str) -> Style:
         style = sfpc(get_style_by_name("solarized-dark"))
 
     return style
+
+
+def signal_handler(sig: int, frame: any) -> None:
+    """
+    Handles Signals (E.g. SIGINT).
+
+    :param sig: Signal
+    :param frame: Signal Frame
+    """
+    if sig == SIGINT:  # If SIGINT - Close application
+        messages = ["Goodbye!", "See you next time!", "Bye bye!"]
+        CONSOLE.print(f"\n\n{choice(messages)}\n")
+        sys.exit(0)
+    else:  # If an unhandled signal is received - Shut down with relevant information.
+        CONSOLE.print(
+            f"\n\n[red]Unexpected signal received - Shutting down.[/red]"
+            f"\nSignal: {sig}\n"
+        )
```

Comment:
Not sure if "Shutting down" seems the write phrase to describe this.

Reply:
Agreed - I see two options really, change the text to something like `Closing piston-cli` or just using one of the farewell messages used in normal shutdown.

Reply:
Lets go with the latter. Probably this:
```suggestion
        CONSOLE.print(f"\n\n[red]Unexpected signal ({sig}) received - {choice(messages)}.[/red]")
```

Reply:
Another thing, we directly just do `import random`;`random.choice()`, personal preference here but it explains its functionality in a much better way.

Reply:
Corrected in latest commit